assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -368,6 +374,17 @@ const PreFareSingleScreenAlert: React.ComponentType<
         />
       );
       break;
+    case effect === "station_closure" && region === "here":
```

Comment:
Out of scope for this change, but it does seem like these effect strings, and possibly region as well, should be in some sort of enum for safety.

Reply:
Agreed, and I think it's a pretty small set of values that can appear here, so I will make that improvement.